src/Command/InstallCommand.php

```diff
@@ -451,11 +466,17 @@ protected function getDatabaseInformation(InputInterface $input, OutputInterface
     protected function verifyDatabaseDetails(array $db, OutputInterface $output)
     {
         $dns = $db['type'] . ':host=' . $db['servers']['write'][0]['host'];
+
         if (isset($db['servers']['write'][0]['port'])) {
             $dns .= ';port=' . (integer)$db['servers']['write'][0]['port'];
         }
+
         $dns .= ';dbname=' . $db['name'];
 
+        if ($db["type"] === "pgsql") {
+            $dns .= ';sslmode=' . $db['servers']['write'][0]['sslmode'];
+        }
+
```

Comment:
When trying to connect via SSL, getting the following error:
```
PHP Censor could not connect to database with the details provided. Please try again.
SQLSTATE[08006] [7] sslmode value "require" invalid when SSL support is not compiled in
```
Code looks correct, will follow up on resolving this issue.

Reply:
This error is Apple's libraries not having SSL support packed in. The connection test succeeded when running in a different environment (Homestead).